### DISCUSSION TITLE
Is there a list of modules and themes developed by the community?

### DISCUSSION THREAD

**Discussion by DrewBrasher** [OrchardCMS/OrchardCore#12957] [Q&A]
It would be nice if before starting to build a theme or module there was a place to go and see if anyone else had made one that you could use instead.

**Comment by hishamco** [comment]
There many you can find in GitHub, these are some to start with:

- [Orchard Core Contrib](https://github.com/OrchardCoreContrib)
- [Lombiq](https://github.com/Lombiq)
- [Surevelox](https://github.com/surevelox/OrchardCore.Modules)

There's a modules, themes and APIs for Orchard Core, may be we can develop a gallery for OC modules /cc @Piedone

**Reply by DrewBrasher** [reply]
Thanks, I will check those out. I think a gallery for OC modules would be great.

**Reply by ns8482e** [reply]
OC Modules are consumed from nuget https://www.nuget.org/packages?q=Tags%3A%22OrchardCoreCMS%22

**Reply by DrewBrasher** [reply]
I would be happy to build or help build a gallery that is easier to browse than nuget.org. I know a great CMS we could use to build it! 😉

**Reply by DrewBrasher** [reply]
I guess google isn't too bad at finding Orchard Core modules. 😆 I was thinking of making a photo gallery to add to my blog site and decided to search to see if anyone had already made one. I found this one:
https://github.com/EtchUK/Etch.OrchardCore.Gallery
https://www.nuget.org/packages/Etch.OrchardCore.Gallery

I haven't tried it yet. Has anyone tried it or any other photo gallery modules?

**Comment by jtkech** [comment]
The Gallery is the Nuget Gallery ;)

Yes there is no dedicated gallery but on Nuget if you search on e.g. `orchardcore fields` it will output `Etch.OrchardCore.Fields` and `StatCan.OrchardCore.ContentFields`.

**Reply by hishamco** [reply]
> The Gallery is the Nuget Gallery ;)

:) but it would be nice if we have Orchard Core gallery
